src/test/java/io/github/christophermanahan/carnitas/HTTPServerTest.java

```diff
@@ -7,73 +7,71 @@
 import java.util.List;
 import java.util.Optional;
 
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-class EchoServerTest {
+class HTTPServerTest {
 
-  private List<String> received;
-  private List<String> sent;
+  private String received;
+  private List<byte[]> sent;
   private Connection connection;
   private Listener listener;
   private TestLogger logger;
 
   @BeforeEach
   void setup() {
-    sent = new ArrayList<>();
+    sent = new ArrayList<byte[]>();
     logger = new TestLogger();
   }
 
   @Test
-  void dataReceivedFromConnectionIsSentToConnection() {
-    received = List.of("data");
-    connection = new TestConnection(received, sent);
+  void sendsHTTPResponseAfterReceivingData() {
+    String data = "data";
```

Comment:
We are using "data" a lot. It is a good idea to test data that is closer to what the real thing is going to be. That affords us two things.

1. More confidence that our code is working.
2. Makes the tests more expressive.